examples/platform/silabs/efr32/rs911x/Rsi91xWifiInterface.cpp

```diff
@@ -61,6 +61,7 @@ extern "C" {
 #include <lib/support/logging/CHIPLogging.h>
 
 #define WFX_QUEUE_SIZE 10
+#define WFX_RSI_BUF_SZ (1024 * 10)
```

Comment:
should we change those to constexpr

Reply:
Yes but i'll do it in a follow up PR when i start refactoring the individual files; I'll fix them all at the same time.